code/game/objects/items/stacks/medical.dm

```diff
@@ -205,6 +205,7 @@
 	icon = 'icons/obj/hydroponics/harvest.dmi'
 	icon_state = "tea_aspera_leaves"
 	color = "#378C61"
+	stop_bleeding = 0

```

Comment:
`FALSE`

Reply:
as a number is fine: \[[source](https://github.com/ParadiseSS13/Paradise/blob/master/code/game/objects/items/stacks/medical.dm#L140)\]